README.md

```diff
@@ -1,27 +1,44 @@
-# SolidusPaypalBraintree
+# SolidusBraintree
 
-[![CircleCI](https://circleci.com/gh/solidusio/solidus_paypal_braintree.svg?style=shield)](https://circleci.com/gh/solidusio/solidus_paypal_braintree)
-[![codecov](https://codecov.io/gh/solidusio/solidus_paypal_braintree/branch/master/graph/badge.svg)](https://codecov.io/gh/solidusio/solidus_paypal_braintree)
+[![CircleCI](https://circleci.com/gh/solidusio/solidus_braintree.svg?style=shield)](https://circleci.com/gh/solidusio/solidus_braintree)
+[![codecov](https://codecov.io/gh/solidusio/solidus_braintree/branch/master/graph/badge.svg)](https://codecov.io/gh/solidusio/solidus_braintree)
 
-`solidus_paypal_braintree` is an extension that adds support for using [Braintree](https://www.braintreepayments.com) as a payment source in your [Solidus](https://solidus.io/) store. It supports Apple Pay, PayPal, and credit card transactions.
+`solidus_braintree` is an extension that adds support for using [Braintree](https://www.braintreepayments.com) as a payment source in your [Solidus](https://solidus.io/) store. It supports Apple Pay, PayPal, and credit card transactions.
 
 🚧 This extension is currently only compatible with the legacy `solidus_frontend` 🚧
 
 ## Installation
 
-Add solidus_paypal_braintree to your Gemfile:
+Add solidus_braintree to your Gemfile:
 
 ```ruby
-gem 'solidus_paypal_braintree', github: 'solidusio/solidus_paypal_braintree', branch: :master
+gem 'solidus_braintree'
 ```
 
 Bundle your dependencies and run the installation generator:
 
 ```shell
 bundle
-bundle exec rails g solidus_paypal_braintree:install
+bundle exec rails g solidus_braintree:install
 ```
 
+## Upgrading to 2.0.0
+
+In v2.0.0, we've renamed the gem to SolidusPaypalBraintree to SolidusBraintree. To update your app:
+
+1. Change the gem in your Gemfile from 'solidus_paypal_braintree' to 'solidus_braintree'.
+2. Run `bin/rails g solidus_braintree:update`. This will run a data migration updating `SolidusPaypalBraintree` references to `SolidusBraintree`.
+3. To support your existing database tables, set the `table_name_prefix` to `'solidus_paypal_braintree_'`:
+
+      ```rb
+      SolidusPaypalBraintree.configure do |config|
+        config.table_name_prefix = 'solidus_paypal_braintree_'
+      end
+      ```
+4. Update the engine route to `mount SolidusBraintree::Engine, at: '/solidus_braintree'`.
```

Comment:
I intentionally avoided automating this with the UpdateGenerator because I avoided allowing the UpdateGenerator to make changes to the user's app code. It can be hard to predict what changes the user has made to their app, and testing for those changes can make the UpdateGenerator more complicated than necessary. Up for discussion :)

Reply:
It's nice having these instructions here, but what about having them also in the installer? I mean printing them in the prompt when the update command is executed.

Reply:
@kennyadsl Hmmm... I'll try first adding a post-install message to the message linking to the README update section. Will let you know how that works out!

Reply:
But users don't need to see them when installing the gem, but after running the update command, or am I missing something?

Reply:
I'm thinking that when they switch from `solidus_paypal_braintree` to `solidus_braintree`, the post-install message would appear. But I'm not sure. Will test it out.